install/update_92_93.php

```diff
@@ -621,6 +621,53 @@ function update92to93() {
 
    /** /Datacenters */
 
+   /** Add address to locations */
+   if (!$DB->fieldExists('glpi_locations', 'address')) {
+      $migration->addField(
+         'glpi_locations',
+         'address',
+         'text COLLATE utf8_unicode_ci',
```

Comment:
you can just use 'text', add migration::fieldFormat will take care of collection and default value